test/TestUserAcceptance.hs

```diff
@@ -939,13 +937,15 @@ sendLiteralKeys keys = do
     waitForString keys defaultCountdown
 
 capture :: ReaderT Env IO String
-capture =
-  tmuxSessionProc "capture-pane"
+capture = T.unpack . decodeLenient . LB.toStrict
```

Comment:
Just as an FYI, `text` provides functions for decoding lazy bytestrings directly.
I don't think we need bother changing to use it here though (i.e. this shouldn't
be a performance issue here, and even if it was it's in the test suite so I don't
care ^_^)

Reply:
Ah cheers! Good to know